bin/oneoff/backfill_data/channel_tokens_storage_app_id

```diff
@@ -11,9 +11,10 @@
   puts "PROCESSING: slice #{slice}..."
   ActiveRecord::Base.transaction do
     channel_tokens_batch.each do |channel_token|
-      _owner_id, channel_id = storage_decrypt_channel_id(channel_token.channel)
-      channel_token.storage_app_id = channel_id
-      channel_token.save!
+      _owner_id, storage_app_id = storage_decrypt_channel_id(channel_token.channel)
+      channel_token.storage_app_id = storage_app_id
+      save_result = channel_token.save(touch: false)
+      raise "ERROR: ID #{channel_token.id}. STORAGE_APP_ID: #{storage_app_id}." unless save_result 
```

Comment:
I found it at http://api.rubyonrails.org/classes/ActiveRecord/Persistence.html#method-i-save. apparently apidock has better SEO but only goes up to rails v4.2.7 whereas we're using rails 5.0.1, and `touch: false` was introduced in rails 5.